tensor2tensor/rl/envs/atari_wrappers.py

```diff
@@ -1,139 +0,0 @@
-# coding=utf-8
```

Comment:
Are we removing the whole file? If not, let's not remove the licence.

Reply:
This file will be deleted as a whole, but we are missing the licence in the other files.